convert2rhel/utils.py

```diff
@@ -560,4 +560,17 @@ def backup(self):
             loggerinst.warning("Can't access %s" % TMP_DIR)
 
 
+def check_if_mount_is_readonly():
+    loggerinst = logging.getLogger(__name__)
```

Comment:
let's merge it after #191 and use logger on the module level? I fixed this in other places

Reply:
Same as my previous comment, #191 can make this change if this PR is merged first. Or this PR can make the change if #191 is merged first

Reply:
#191 merged. @fellipeh, please rebase and use logger defined at the module level.